controller/src/test/java/io/pravega/controller/store/stream/StreamMetadataStoreTest.java

```diff
@@ -1971,14 +1971,18 @@ public void testReaderGroups() throws Exception {
                 .startingStreamCuts(startSC)
                 .endingStreamCuts(endSC).build();
         final RGOperationContext rgContext = store.createRGContext(scopeRGTest, rgName);
-        store.addReaderGroupToScope(scopeRGTest, rgName, rgConfig.getReaderGroupId());
+        store.addReaderGroupToScope(scopeRGTest, rgName, rgConfig.getReaderGroupId()).join();
         store.createReaderGroup(scopeRGTest, rgName, rgConfig, System.currentTimeMillis(), rgContext, executor).join();
+        UUID readerId = store.getReaderGroupId(scopeRGTest,rgName).get();
```

Comment:
nit: please call this readerGroupId as readerId is a separate concept and can be misleading for anyone reading the code.

Reply:
done